source/polling-srv-records-for-mongos-discovery/polling-srv-records-for-mongos-discovery.rst

```diff
@@ -141,65 +141,6 @@ scanning_ functionality, but only if *rescanSRVIntervalMS* has passed.
 
 .. _scanning: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#scanning
 
-Test Plan
-=========
```

Comment:
I think this section should remain with a link to the tests directory.

See: https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst#test-plan

Reply:
Made this change.